internal/commands/app/diff_test.go

```diff
@@ -292,4 +342,19 @@ func TestAppDiffInputs(t *testing.T) {
 			tc.test(t, tc.inputs, profile)
 		})
 	}
+
+	t.Run("should return error with include node modules and include package json are both set", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+
+		_, console, _, ui, consoleErr := mock.NewVT10XConsole()
```

Comment:
avoid using the `V10XConsole` unless you're writing tests that require interactivity/pseudo-terminals

this can just use `_, ui := mock.NewUI()`

Reply:
yes, and this will be reflected on the test above, this test is duplicate.